packages/video_player/video_player/lib/video_player.dart

```diff
@@ -763,6 +792,14 @@ class VideoPlayerController extends ValueNotifier<VideoPlayerValue> {
     Future<ClosedCaptionFile>? closedCaptionFile,
   ) async {
     _closedCaptionFile = await closedCaptionFile;
+
+    _sortedCaptions = _closedCaptionFile?.captions;
+
+    /// Sort the captions by start time so that we can do a binary search.
```

Comment:
"so that we can do" -> "to allow"

Using "we" in comments is often actively confusing for readers, because who "we" is isn't usually clear.